ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController.swift

```diff
@@ -248,11 +247,66 @@ class NotificationViewDemoController: DemoController {
             preconditionFailure("showNotificationView is used for a button in the wrong container")
         }
 
-        createNotificationView(forVariant: variant).show(in: view) { $0.hide(after: 3.0) }
+        let notification = createNotificationView(forVariant: variant)
+        if variant == .warningToastWithFlexibleWidth {
+            showNotification(notification: notification)
+        } else {
+            notification.show(in: view) { $0.hide(after: 3.0) }
+        }
     }
 
     @objc private func showSwiftUIDemo() {
         navigationController?.pushViewController(NotificationViewDemoControllerSwiftUI(),
                                                  animated: true)
     }
+
+    private func showNotification(notification: MSFNotification) {
```

Comment:
so bascially we need to update the show an documentation that client shouldn't use it if they want flexiblewidth? what is a good way to calculate the best estimate of swiftuinotfication instrinsic size to uikit callers?